src/server/game/Entities/Player/Player.cpp

```diff
@@ -4323,6 +4323,49 @@ void Player::DeleteOldCharacters(uint32 keepDays)
     }
 }
 
+/**
+ * Items which were kept back in the database after being deleted and are now too old (see config option "ItemDelete.KeepDays"), will be completely deleted.
+ */
+void Player::DeleteOldItems()
```

Comment:
What do you think about renaming this and others to something like `DeleteOldRecoveryItems`? Without context this function definition may be misleading.